cfme/utils/bz.py

```diff
@@ -207,6 +209,58 @@ def resolve_blocker(self, blocker, version=None, ignore_bugs=None, force_block_s
                 return bug
         return None
 
+    def set_flags(self, idlist, flags):
+        bz_creds = credentials.get("bugzilla", None)
+
+        # login to bzapi
+        if not bz_creds:
+            raise BugzillaError("No bugzilla creds available")
+        try:
+            self.bugzilla.login(bz_creds.get("username", None), bz_creds.get("password", None))
+        except BugzillaError:
+            logger.exception("Unable to login to Bugzilla with those creds.")
+            raise
+
+        # set the flags
+        for bug in self.bugzilla.getbugs(idlist):
+            result = bug.updateflags(flags)
+            logger.info("Got %s from updating %s", result, bug)
+
+        # logout
+        self.bugzilla.logout()
+
+    def get_bz_info(self, idlist):
+        bz_creds = credentials.get("bugzilla", None)
+
+        # login to bzapi
+        if not bz_creds:
+            raise BugzillaError("No bugzilla creds available")
+        try:
+            self.bugzilla.login(bz_creds.get("username", None), bz_creds.get("password", None))
```

Comment:
Required:
default is `None` if not found "username".

Reply:
Oh yeah, good point, I'll remove that